### PR TITLE
Add a example for Calling custom MATLAB function

### PR DESCRIPTION
**Pull request by SHIELD-SKY**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -310,6 +310,38 @@ julia> @show y
...
+
+then we can call it in Julia file:
+
+```julia
+using MATLAB
+
+x = range(-10.0, stop=10.0, length=500)
+y = range(2.0, stop=3.0, length=500)
+
+mat"addpath('path of folder')"
```
Comment (musm) [inline README.md:338]:
```suggestion
mat"addpath('path/to/folder')"
```
?

Comment (SHIELD-SKY) [reply README.md]: Thank your very much for your advices. This is my first time to  make a PR, I learned a lot from your suggestions. I have revised  the file.

File: README.md [hunk 2]
```diff
@@ -310,6 +310,38 @@ julia> @show y
 a = 63.0
 ```
 
+#### Calling custom MATLAB function
+
+Adding the location of the function to path, then call the function:
+
+```julia
+mat"addpath('path/to/folder')"
```
Comment (musm) [inline README.md:318]:
```suggestion
mat"addpath('/path/to/folder')"
```

**Comment by musm** [comment]
thanks!

**Pull request merged**